frontend/app/model/Eventbrite.scala

```diff
@@ -46,24 +46,28 @@ object Eventbrite {
     lazy val blurb = truncateToWordBoundary(text, 120)
   }
 
-  case class EBAddress(address_1: Option[String],
-                       address_2: Option[String],
-                       city: Option[String],
-                       region: Option[String],
-                       postal_code: Option[String],
-                       country: Option[String]) extends EBObject {
-
+  case class EBAddress(
```

Comment:
Yeah, this is existing code but I think the reason is because it's mapped directly from the EventBrite API and these values are all optional there.